.github/workflows/database-backup.yml

```
@@ -105,20 +105,41 @@ jobs:
       env:
         SQLPAD_USER: ${{ secrets.SQLPAD_USER }}
 
-    - name: Sanitise the Database backup
+    - name: Disk cleanup
+      shell: bash
+      run: |
+        sudo rm -rf /usr/local/lib/android || true
+        sudo rm -rf /usr/share/dotnet || true
+        sudo rm -rf /opt/ghc || true
+
+    - name: Create local Sanitised Database
       run: |
-        echo "::group::Restore backup to intermediate database"
         createdb ${DATABASE_NAME} && psql -f ${{ env.BACKUP_FILE_NAME }}.sql -d ${DATABASE_NAME}
+      env:
+        DATABASE_NAME: register_trainee_teachers
+        PGUSER:  postgres
+        PGPASSWORD: postgres
+        PGHOST: localhost
+        PGPORT: 5432
+
+    - name: Remove backup file
+      shell: bash
+      run: |
         rm ${{ env.BACKUP_FILE_NAME }}.sql
-        echo "::endgroup::"
 
-        echo "::group::Sanitise user data"
+    - name: Sanitise the local Database
+      run: |
         psql -d ${DATABASE_NAME} -f db/scripts/sanitise.sql
-        echo "::endgroup::"
+      env:
+        DATABASE_NAME: register_trainee_teachers
+        PGUSER:  postgres
+        PGPASSWORD: postgres
+        PGHOST: localhost
+        PGPORT: 5432
 
-        echo "::group::Backup Sanitised Database"
+    - name: Dump the Sanitised Database
+      run: |
         pg_dump --encoding utf8 --compress=1 --clean --no-owner --if-exists -d ${DATABASE_NAME} -f backup_sanitised.sql.gz
```

Maybe compress some more?

```suggestion
        pg_dump --encoding utf8 --compress=9 --clean --no-owner --if-exists -d ${DATABASE_NAME} -f backup_sanitised.sql.gz
```
With 9 being the max

or 
```suggestion
        pg_dump --encoding utf8 --compress=6 --clean --no-owner --if-exists -d ${DATABASE_NAME} -f backup_sanitised.sql.gz
```

With 6 being the underlaying default

---

We ran some tests a while back, and you didn't get much improvement on disk saved vs time taken when you increased the compression level. It currently compresses to 276MB which is pretty good. Would be worth increasing the level if it was a lot bigger. It also turned out this step wasn't really the issue (sanitised backup creation), it was failing on the previous step when it tried to create a local postgres db with the original backup.

---

Ok